libcontainer/criu_opts_linux.go

```diff
@@ -1,7 +1,5 @@
 package libcontainer
```

Comment:
Is it possible to also move the `CriuPageServerInfo` stuff from `checkpoint.go` so we can mark this file as `//go:build !runc_nocr` as well? I guess it doesn't matter for the size of the final binary, but it is a little odd to keep this even with `runc_nocr`.

Actually, would it be possible to move `criuOptions` to a `!runc_nocr` file, or would that be too difficult?

Reply:
I did that initially, but when I realized that `runc checkpoint` and `runc restore` commands should be kept intact, so when a user runs those, they see a useful error like "c/r not compiled in" rather than `No help topic for 'checkpoint'` (which is urfave/cli way to say no such command).

Same for all the checkpoint/restore options: I kept those so that a valid command like `runc checkpoint --leave-running XXX` results in a useful "c/r not compiled in" error rather than "flag provided but not defined: -leave-running`.

In order to keep both commands and their options, we have to keep the data structures, too. We're lucky there aren't that many.

Sure, we can duplicate the whole thing with commands and options but without data structures, but that would be a mere duplication.

So, for the sake of both simplicity and user experience I keep all the CLI parsing (and the associated data structures) in place. Hope it makes sense.